src/main/resources/assets/computercraft/lua/rom/motd.txt

```diff
@@ -3,3 +3,13 @@ View the documentation at https://wiki.computercraft.cc
 Visit the forum at https://forums.computercraft.cc
 You can disable these messages by running "set motd.enable false"
 You can create directories with "mkdir".
+Want to see hidden files? Run "set list.show_hidden true".
+Run "list","ls" or "dir to see all files in a directory.
+You can delete files and directories with "delete" or "rm".
+Use "pastebin put" to upload a program to pastebin.
+Use "pastebin get" to download a program from pastebin.
+Use "pastebin run" to run a program from pastebin without downloading it.
+If you using a pocket computer, you can play tetris with the "falling" program.
+You can use the "paint" program to draw pictures.
```

Comment:
Can we remove this and the tetris one for now? Ideally you'd only show these when the program is actually available (which they won't be on normal computers, etc...).

Reply:
If we only show this, when it's available, we need to create a extra motd for pocket and for advanced computers.

Reply:
Yeah, that's why I was just suggesting removing it for now - short of making motd files dynamic, there's not really a nice way of restricting which ones we show.

Reply:
I just think, it won't hurt, if people who are using a normal computer see what they can do with a advanced computer.

Reply:
I think the tetris one is too long anyway with the extra sentence regarding the pocket computer, so it should be removed until/if there is support for per-device motd